worker/src/utils/validate.ts

```diff
@@ -30,9 +30,9 @@ export const IncomingPayload = object({
     array(
       object({
         slug: string(),
-        protected: boolean(),
+        protected: nullable(boolean()),
```

Comment:
Should we default it to the default value then?

Reply:
I looked at the supervisor source, `None` is the default

Reply:
But that would be the same as `True` here right?

Reply:
OK, so use `create` instead of `assert`? 😄